datalad/support/gitrepo.py

```diff
@@ -2111,7 +2118,7 @@ def _git_custom_command(self, files, cmd_str,
                             log_stdout=True, log_stderr=True, log_online=False,
                             expect_stderr=True, cwd=None, env=None,
                             shell=None, expect_fail=False,
-                            check_fake_dates=False,
+                            read_only=False,
                             index_file=None,
```

Comment:
`_git_custom_command` is no longer used in the code base.  I think it'd be better to just leave it be.

Reply:
I am ok either way (since likely to be removed if not used I guess).  Let's see if any other changes needed and I will adjust

Reply:
ok, added reverting commit